aggregators/aggregate.js

```diff
@@ -54,6 +78,13 @@ class Aggregate {
     return remapped;
   }
 
+  /**
+   * Converts each raw audit output to a weighted value for the aggregation.
+   * @private
+   * @param {!AuditResult} result The audit's output value.
+   * @param {!AggregationCriterion} expected The aggregation's expected value and weighting for this result.
+   * @return {number} The weighted result.
+   */
   static _convertToWeight(result, expected) {
     let weight = 0;
 
```

Comment:
nit on the next line as it relates to types - `typeof expected === 'undefined'` only returns true if undefined is passed, which shouldn't happen because of your Closure annotation. If the aim is to be defensive then a `!expected` check might just be easier.